### DISCUSSION TITLE
Welcome to nodestream Discussions!

### DISCUSSION THREAD

**Discussion by zprobst** [nodestream-proj/nodestream#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by travishaby** [comment]
👋 Giving this framework a go as a replacement for a homespun ETL process that takes files from S3, creates entities + relationships, and then loads them into a Neptune DB. Appreciate all y'all's work here!

**Reply by ccloes** [reply]
Welcome to the party @travishaby ... let us know if you have any questions.
